iFixit/react-commerce
# Custom slider implementation

We were using swiper.js to implement sliders in the codebase.
Since the bundle size impact was relevant even if we were using just a few of the functionalities it included, we made a custom slider implementation.

### QA

1. Visit Vercel preview
2. Verify that all 3 slider usages (product gallery in product page + press and quotes section in the store homepage) are still looking and working as before.
